README.md

```diff
@@ -23,7 +23,12 @@ preprocessing results ensures reasonable performance.
 
 ## Installation: ##
 ----------
-- For **deployers** (i.e. if you want to primarily *use* the system as opposed to modify/develop it, the [docker installation](https://github.com/e-mission/e-mission-docker) is probably the easiest way to get started.
+- For **deployers** i.e. if you want to primarily *use* the system as opposed to modify/develop it,
+
+```bash
+docker build -t e-mission-server .
+```
+
```

Comment:
I am also not sure why you are building this container instead of using one of the pre-built images. We build an image on every commit to master.
https://github.com/e-mission/e-mission-server/actions/workflows/image_build_push.yml